documentation/README.md

```diff
@@ -84,39 +88,84 @@ To maximize screen real estate, Music Blocks overlays the program
 elements (stacks of blocks) on top of the canvas. These blocks can be
 hidden at any time while running the program.
 
+
 ## <a>2. Toolbars</a>
 
 [Back to Table of Contents](#table-of-contents)
 
-There are five toolbars:
 
-(1) The *Main* toolbar across the top of the screen. There you will
-find the *Play* button, the *Stop* button, the *New Project* button,
-buttons for loading and saving projects, the *Planet* button, where
-you can access community projects, the *hamburger* button, which opens
-the secondary toolbar, and the *help* button.
+There are five toolbars in Music Blocks, each serving a specific purpose to help you navigate and interact with the application. Below is a detailed description of each toolbar along with images to visually illustrate their functions:
+
+### 1. Main Toolbar
+
+The **Main** toolbar is located across the top of the screen. It contains several essential buttons for managing your project, controlling playback, and accessing various settings.
+
+-   **Play Button**: Starts playing the project.
+-   **Stop Button**: Stops the current project playback.
+-   **New Project Button**: Creates a new project from scratch.
+-   **Load Project from File Button**: Opens an existing project file.
+-   **Save Project Button**: Saves the current project.
+-   **Find and Share Project Button**: Opens the Planet view where you can explore and share projects with the community.
+-   **Auxiliary Menu**: Provides additional tools for project settings and configuration.
+-   **Help Button**: Opens the help menu, providing access to instructions and guidance.
+
+---
+
+### 2. Secondary Toolbar
+
+The **Secondary** toolbar appears when you click the **hamburger button** (three horizontal lines) in the main toolbar. This toolbar provides several options to control the execution of your project and adjust your work environment.
```

Comment:
The "beginner" mode is as you describe it here. The "advanced" mode has many more buttons.